packages/upset/src/components/SvgBase.tsx

```diff
@@ -27,7 +25,7 @@ export const SvgBase: FC<Props> = ({ children, defaultSettings }) => {
         height: 100%;
         width: 100%;
         text.icon {
-          font-family: 'Font Awesome 5 Free';
+          font-family: Arial, sans-serif;
           font-weight: 700;
           font-size: 12px;
         }
```

Comment:
Remove?

Reply:
Resolved in [bae2b9f](https://github.com/visdesignlab/upset2/pull/361/commits/bae2b9fcb606e3b81d781486a406bc9c2bb1c338)